src/app/components/client/ExposureCard.tsx

```diff
@@ -134,6 +137,95 @@ const ScanResultCard = (props: ScanResultCardProps) => {
   const isCompanyNameTooLong =
     scanResult.data_broker.length > COMPANY_NAME_MAX_CHARACTER_COUNT;
 
+  const dataBrokerProfileLink = (
+    <TelemetryLink
+      href={scanResult.link}
+      target="_blank"
+      eventData={{
+        link_id: `data_broker_${scanResult.id}`,
+      }}
+    />
+  );
+
+  const removalInfoLink = (
+    <TelemetryLink
+      href={CONST_URL_SUMO_MONITOR_REMOVAL}
+      target="_blank"
+      eventData={{
+        link_id: "explanation_of_removal_time",
+      }}
+    />
+  );
+
+  const upsellLink = (
+    <TelemetryLink
+      upsell
+      eventData={{
+        // TODO: Check that this attribution id is accurate
+        link_id: "clicked_upsell",
+      }}
+      href="/user/dashboard/fix/data-broker-profiles/automatic-remove"
+    />
+  );
+
+  const dataBrokerDescription = () => {
+    switch (scanResult.status) {
+      case "optout_in_progress":
+      case "waiting_for_verification":
+        return l10n.getFragment(
+          "exposure-card-description-info-for-sale-in-progress",
+          {
+            elems: {
+              data_broker_profile: dataBrokerProfileLink,
+              removal_info: removalInfoLink,
+            },
+          },
+        );
+      case "new":
+        // Data broker cards manually resolved do not change their status to "removed";
+        // instead, we track them using the "manually_resolved" property.
+        if (scanResult.manually_resolved) {
+          return l10n.getFragment(
+            "exposure-card-description-info-for-sale-fixed-manually-fixed",
+            {
+              elems: {
+                data_broker_profile: dataBrokerProfileLink,
+              },
+            },
+          );
+        }
+        if (isOnManualRemovePage) {
+          return l10n.getFragment(
+            "exposure-card-description-info-for-sale-action-needed-manual-fix-page",
+            {
+              elems: {
+                data_broker_profile: dataBrokerProfileLink,
+                upsell_link: upsellLink,
+              },
+            },
+          );
+        }
+        return l10n.getFragment(
+          "exposure-card-description-info-for-sale-action-needed-dashboard",
+          {
+            elems: {
+              data_broker_profile: dataBrokerProfileLink,
+              upsell_link: upsellLink,
+            },
+          },
+        );
+      case "removed":
+        return l10n.getFragment(
+          "exposure-card-description-info-for-sale-fixed",
+          {
+            elems: {
+              data_broker_profile: dataBrokerProfileLink,
+            },
+          },
+        );
+    }
```

Comment:
Does this need a `default` case to catch any unexpected `scanResult.status` values? (or is that already covered elsewhere by TypeScript types?)